featurebyte/routes/feature_store/controller.py

```diff
@@ -127,27 +121,23 @@ async def _updated_get_credential(user_id: str, feature_store_name: str) -> Any:
                 Any
                     credentials
                 """
-                cred = await get_credential(user_id, feature_store_name)
```

Comment:
the main change is to remove this check as `get_credential` shouldn't be passing anything in anymore.